src/components/Home/ClusterDashboardItem.js

```diff
@@ -25,6 +25,21 @@ import ClusterDashboardResourcesV4 from './ClusterDashboardResourcesV4';
 import ClusterDashboardResourcesV5 from './ClusterDashboardResourcesV5';
 import UpgradeNotice from './UpgradeNotice';
 
+const LabelWrapper = styled.div`
+  width: 90px;
```

Comment:
I believe this would be more future-proof. What do you think?
```suggestion
  min-width: 90px;
  max-width: 100%;
```

Reply:
Well, I don't know... In which scenarios do you think it would be useful?

Reply:
In case the ID get longer (which will happen, since we'll let users set their own IDs in the near future"